core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -5,7 +5,7 @@
         <ul class="author-profile">
           <li>
             <img ng-src="<[getStaticImageUrl('/general/apple.svg')]>" class="exploration-footer-img"
-                 ng-if="contributorNames.length > 0" alt="">
+                alt="">
```

Comment:
Good catch, but indent this line by 1 more space so the left border of "alt" matches that of "ng-src".